README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
-[] Arrays
-[] Null
+[X] Integers
+[X] Arrays
```

Comment:
In JavaScript, an array is a special type of object. Therefore typeof [1,2,3,4] returns object.